src/algebra/classical_lie_algebras.lean

```diff
@@ -149,6 +149,12 @@ bilinear form defined by the identity matrix. -/
 def so : lie_subalgebra R (matrix n n R) :=
   skew_adjoint_matrices_lie_subalgebra (1 : matrix n n R)
 
+lemma mem_so (A : matrix l l R) : A ∈ so l R ↔ Aᵀ = -A :=
```

Comment:
I don't know this corner of the library, but should this be `@[simp]`, or is it better to leave things in terms of `so`?

Reply:
I was 50-50 myself. Your comment has pushed me over the edge so I've added it.